src/__tests__/utils/feedbackGif.test.ts

```diff
@@ -0,0 +1,22 @@
+import { getRandomGifSrc } from '../../utils/feedbackGif';
+
+const arr = [0, 1, 3];
```

Comment:
the array passed into this function is meant to be a string[] - it's my bad for not adding proper types to the function's header (add them now pls)

It's supposed to be an array of filepaths like `['1.gif', '2.gif', '3.gif']`